projects/canopy/src/styles/utils.scss

```diff
@@ -3,3 +3,24 @@
 .lg-visually-hidden {
   @include lg-visually-hidden;
 }
+
+$breakpoints: (
```

Comment:
Do these also still exist in the mixins.scss file? (which is arguably an odd place for them originally) I just wanted to make sure they weren't repeated.

Reply:
Ah yes they do - was meant to remove this stuff as it was an early thing - I've removed this bit